src/ripple/app/misc/impl/AccountTxPaging.cpp

```diff
@@ -68,8 +68,8 @@ accountTxPage(
     std::function<void(
         std::uint32_t,
         std::string const&,
-        Blob const&,
-        Blob const&)> const& onTransaction,
+        Blob&&,
+        Blob&&)> const& onTransaction,
```

Comment:
No action needed; just admiring the `accountTxPage` signature... 12 arguments, two of which are callbacks. :grimacing: